services/server/src/server/services/utils/database-util.ts

```diff
@@ -632,3 +636,38 @@ export async function getSourcifyMatchAddressesByChainAndMatch(
     [chain, page * paginationSize, paginationSize]
   );
 }
+
+export async function updateContract(
+  pool: Pool,
+  { id, creation_bytecode_hash, runtime_bytecode_hash }: Tables.Contract
+) {
```

Comment:
Maybe add a type annotation here for the returning value. Should be `Tables.Contract`.

Reply:
After a closer look, I saw this function is not used. Any reason for adding it here? Maybe for the fixing script?